src/Internal/Contract/Monad.purs

```diff
@@ -257,11 +248,11 @@ getLedgerConstants
        , systemStart :: Ogmios.SystemStart
        }
 getLedgerConstants logger = case _ of
-  CtlBackend { ogmios: { ws } } -> do
+  CtlBackend { ogmios: { ws } } _ -> do
     pparams <- getProtocolParametersAff ws logger
     systemStart <- getSystemStartAff ws logger
     pure { pparams, systemStart }
-  BlockfrostBackend _ -> undefined
+  BlockfrostBackend _ _ -> undefined
```

Comment:
Will this `undefined` not be an issue? For example [here](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1314/files#diff-cd9604d9a86ee84c0def99f84ddea1aa930a7b511a2ada6493fdea9e61ced7cfR190)

Reply:
I think it's okay as long as we don't export to the `BlockfrostBackend` to public interface yet. After all the point of #1119  is to  be able to work this in #1120

Reply:
Ah I see, thanks for the info